src/toil/batchSystems/mesos/batchSystem.py

```diff
@@ -572,6 +572,12 @@ def executorLost(self, driver, executorId, slaveId, status):
         log.warning("Executor '%s' lost.", executorId)
 
 
+    @classmethod
+    def setOptions(cl, config):
+        super.setOptions(config)
```

Comment:
Sorry - lack of familiarity with Python. I've pushed a fix for this. It looks like the PR updates automatically. 

I don't see jenkins output.